static/app/views/nav/secondary/sections/codecov/codecovSecondaryNav.tsx

```diff
@@ -26,7 +28,7 @@ function CodecovSecondaryNav() {
   });
 
   return (
-    <SecondaryNav>
+    <Fragment>
```

Comment:
Moved this `SecondaryNav` wrapper from the `*SecondaryNav` components to a single parent component since we do not want it in the overlay.